lib/runner/extensions/event.command.js

```diff
@@ -183,33 +257,48 @@ module.exports = {
                     });
 
                 this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId, function (cursor, id, requestId, request) {
-                    // clone cursor and add source
-                    var cursorWithSource = _.clone(cursor),
+                    // remove files in request body if any
+                    sanitizeFiles(request, function (err, sanitizedRequest, affectedEntries) {
```

Comment:
call this parameter `request` itself so that it overrides the outer closure variable in request and prevent bigs.